apps/rush-lib/src/logic/taskExecution/ProjectTaskRunner.ts

```diff
@@ -54,7 +54,7 @@ export interface IProjectTaskRunnerOptions {
   projectChangeAnalyzer: ProjectChangeAnalyzer;
   allowWarningsInSuccessfulBuild?: boolean;
   taskName: string;
-  phase: IPhase;
+  phase: Pick<IPhase, 'name' | 'logFilenameIdentifier'>;
```

Comment:
Why the computed type?

Reply:
This had been done because I was translating the phase type in ProjectTaskSelector, but is now no longer necessary.